API/chat/views.py

```diff
@@ -7,12 +7,15 @@
 
 from .models import Channel, Message
 from .forms import MessageCreationForm, MessagePatchForm
+from django.conf import settings
 
 
 class MessageView(View):
     def post(self, request, type, target, *args, **kwargs):
         # currently `type` is always 'channel'
         channel = get_object_or_404(Channel, name=target)
+        if not settings.PASS == request.META['HTTP_PASSWORD']:
+            return HttpResponseBadRequest()
```

Comment:
This is not a bad request. Please use the appropriate HTTP header - [401 Unauthorized](http://stackoverflow.com/a/6937030/126342)